용재/1406.py

```diff
@@ -0,0 +1,27 @@
+# https://www.acmicpc.net/problem/1406
+# 에디터
+
+getText = list(input())
+
+N = int(input())
+
+inputNumbers = [str(input()) for _ in range(N)]
```

Comment:
```suggestion
inputNumbers = [input().split() for _ in range(N)]
```

string 그대로 입력받지 말고 split해서 넣어주는 것은 어떨까요?